spec/rest-middleware.spec.js

```diff
@@ -0,0 +1,84 @@
+"use strict";
+
+describe("restMiddleware", () => {
+    var configMock, helmetMock, logger, restMiddleware, serverMock, restifyLinks;
+
+    beforeEach(() => {
+        var LoggerMock, RestMiddleware;
+
+        RestMiddleware = require("../lib/rest-middleware");
+        LoggerMock = require("./mock/logger-mock");
+        helmetMock = jasmine.createSpyObj("helmetMock", [
+            "frameguard",
+            "hidePoweredBy",
+            "hsts",
+            "ieNoOpen",
+            "noCache",
+            "noSniff",
+            "xssFilter"
+        ]);
+        serverMock = jasmine.createSpyObj("serverMock", [
+            "use"
+        ]);
+        configMock = {
```

Comment:
A mocked object isn't a value-only object.  You should not call this one a mock because there's no methods on it.